<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Login_LoginButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_Quit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Login_UserNumberEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Login_UserPasswordEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Login_UserNumber_Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>员工号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Login_UserPasswordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>461</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Login_LoginButton</zorder>
  <zorder>Login_Quit</zorder>
  <zorder>Login_UserNumberEdit</zorder>
  <zorder>Login_UserPasswordEdit</zorder>
  <zorder>Login_UserNumber_Label</zorder>
  <zorder>Login_UserPasswordLabel</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
